styles/pages/statistics.module.scss

```diff
@@ -0,0 +1,18 @@
+.main {
```

Comment:
The current setup, while works, is not desirable. It's preferable that the `main` element will have a consistent shared style across the site, and if there is a need for a different layout than override it per page.
For example you wouldn't want to declare every new page:
```scss
flex-grow: 1;
```

As I see it there are 2 options: 
1. Every page add the overriding declartions to `main`.
2. Keep `main` untouched and every page give the list a container for it (here for example a parent container would be `.stats`).

I think 2 is better, but your choice. 
Either way declare globally the things that are shared between pages

Reply:
noted